docs/develop/sdk/1-dev-setup.md

```diff
@@ -0,0 +1,161 @@
+2022-05-05 13:25:14 📦 Highest known block at #0
+2022-05-05 13:25:14 〽️ Prometheus exporter started at 127.0.0.1:9615
+2022-05-05 13:25:14 Listening for new connections on 0.0.0.0:9944.
+2022-05-05 13:25:19 💤 Idle (0 peers), best: #0 (0x09fc…3a2b), finalized #0 (0x09fc…3a2b), ⬇ 0 ⬆ 0
+2022-05-05 13:25:20 Accepted a new tcp connection from 172.17.0.1:56636.
+2022-05-05 13:25:23 🙌 Starting consensus session on top of parent 0x...
+2022-05-05 13:25:23 🎁 Prepared block for proposing at 1 (3 ms) [hash: 0x...; parent_hash: 0x09fc…3a2b; extrinsics (1): [0xae1a…0701]]
+2022-05-05 13:25:23 🔖 Pre-sealed block for proposal at 1. Hash now 0x..., previously 0x....
+```
+
+Congratulations!
+You are running your own blockchain. 🎉
+
+The blockchain exposes a websocket endpoint on port `9944` and an RPC endpoint on port `9933`.
+You can test that by first calling an RPC endpoint using curl.
+
+```
+curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "system_name", "params":[]}' http://127.0.0.1:9933/
+```
+
+This should give you `{"jsonrpc":"2.0","result":"KILT Node","id":1}` as a response.
+
+But you can also connect to the websocket endpoints using our little script above.
+Simple replace the Peregrine address `wss://peregrine.kilt.io/parachain-public-ws` with `ws://127.0.0.1:9944`.
+Note that we connect to the port `9944` as we are using the websocket protocol for our SDK and not bare http.
+
+
+## Standalone vs. Parachain (Peregrine/Spiritnet)
+
+The standalone chain is very close in functionality to our parachains but there are a few fundamental differences between them.
+
+<!-- ### Blocktime
+
+Block time is actually the same, but this might change in the future. -->
+
+### Governance
+
+While governance is a very important part of our parachains, it is not used in the standalone version and replaced by the sudo pallet.
+None of the following pallets are part of the standalone chain, but they are all part of the parachain runtime:
+
+* Democracy
+* Council
+* TechnicalCommittee
+* TechnicalMembership
+* Treasury
+* Scheduler
+
+### Staking
+
+Staking is a feature that is used to elect who is allowed to produce blocks.
+It is important for parachains to have this election as decentralized as possible.
+But for a local development chain it is not necessary since you will be the only one producing blocks.
+
+### Deployment Complexity
+
+Deploying a parachain is more complext than deploying a standalone chain.
+For the standalone node, you simple execute a single docker command.
+The task of spinning up a parachain can be split up in three steps.
+
+1. Setup a relay chain with 4 validators.
+2. Start and connect your parachain node to the relaychain.
+3. Register your parachain using the runtime WASM and the genesis state.
+
+Since these steps are not trivial to execute and take quite some time to do by hand, we started a [docker based setup](https://github.com/KILTprotocol/local-parachain-setup) which automates these steps.
```

Comment:
```suggestion
Since these steps are not trivial to execute and take quite some time to do by hand, we provide a minimal [docker based setup](https://github.com/KILTprotocol/local-parachain-setup) which automates these steps.
```